subprojects/gst-docs/examples/tutorials/.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "C_Cpp.default.compileCommands": "/home/pgarcia/fluendo/gstreamer/subprojects/gst-docs/examples/tutorials/builddir/compile_commands.json",
```

Comment:
This is an absolute path from your personal computer. Would it be possible to use a variable or a relative path here?

Reply:
Done

Reply:
Missing `git push` maybe?